raw-spans-grouper/src/test/java/org/hypertrace/core/rawspansgrouper/RawSpanToStructuredTraceGroupingJobConfigTest.java

```diff
@@ -6,12 +6,20 @@
 import com.typesafe.config.ConfigException;
 import com.typesafe.config.ConfigFactory;
 import java.io.File;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public class RawSpanToStructuredTraceGroupingJobConfigTest {
   @Test
   public void testMinimalCompleteConfig() {
-    createRawSpansGrouperJobUsingConfig("minimal-complete-config.conf");
+    RawSpanToStructuredTraceGroupingJob job = createRawSpansGrouperJobUsingConfig("minimal-complete-config.conf");
+    Assertions.assertEquals(Runtime.getRuntime().availableProcessors(), job.getExecutionEnvironment().getParallelism());
```

Comment:
Not sure we should be testing what flink sets the default to, just that it's some (presumably nonzero) value.

Reply:
Actually the code that set's it to number of CPUs in node is our code too but in the flink-utils dependency.

Reply:
I see it in the flink code too, are we overriding that with our own default? 🙄  - Either way, shouldn't be tested here.

Reply:
The default without calling `StreamExecutionEnvironment.createLocalEnvironment(Runtime.getRuntime().availableProcessors(), configuration)` is -1. So are you saying this should be an assert on > 0 and not match the exact expected value?

Reply:
OK actually looked at the code now - so our code calls the flink code and since we provide a config, we need to explicitly set parallelism inside the flink util code.

Since that code is already called in this test, it should already be greater than 0 - so I'd just assert that parallelism is greater than 0. This test shouldn't know or care what that default is or how it gets set.

Reply:
done.